yellowbox/extras/logstash.py

```diff
@@ -18,6 +18,23 @@
 _CLOSE_SENTINEL = b"\0"
 
 
+def _level_to_int(level: Union[str, int]) -> int:
+    """Convert a log level in str or int into an int.
+
+    Args:
+        level: Can be a string or int
+
+    Returns:
+        level as int.
+    """
+    if not isinstance(level, int):
+        level = logging.getLevelName(level.upper())
+
+    # In rare cases it might be false, but it shouldn't generally happen.
+    assert isinstance(level, int)
```

Comment:
Looking at the source code of `getLevelName`, it would be pretty easy to get a non-int here by misspelling a level name. Raise a better exception.